backend-elasticsearch/src/main/java/org/hibernate/search/v6poc/backend/elasticsearch/document/model/impl/ElasticsearchIndexCollectorImplMetadata.java

```diff
@@ -28,8 +28,8 @@
 /**
  * @author Yoann Rodiere
  */
-public class ElasticsearchIndexSchemaCollectorImpl<T extends TypeMapping>
-		implements ElasticsearchIndexSchemaElement, IndexSchemaCollector,
+public class ElasticsearchIndexCollectorImplMetadata<T extends TypeMapping>
+		implements ElasticsearchIndexElementMetadata, IndexCollectorMetadata,
```

Comment:
Shouldn't it be `IndexMetadataCollector`?

Reply:
Seems outdated? I renamed it in a later commit.